tutorials/workflow/csharp/fundamentals/Basic/Program.cs

```diff
@@ -0,0 +1,24 @@
+using Dapr.Workflow;
+using Basic;
+using Basic.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprWorkflow(options => {
+    options.RegisterWorkflow<BasicWorkflow>();
+    options.RegisterActivity<Activity1>();
+    options.RegisterActivity<Activity2>();
+});
+var app = builder.Build();
+
+app.MapPost("/start/{input}", async (string input) => {
+    await using var scope  = app.Services.CreateAsyncScope();
+    var workflowClient = scope.ServiceProvider.GetRequiredService<DaprWorkflowClient>();
```

Comment:
Nope, as in a previous message- this isn't necessary. Inject `DaprWorkflowClient` into the async method (can use an optional `[FromServices]` attribute if you want) and access the instance from there. A separate scope is created by invocation of the inbound post request itself and isn't needed here.